### ISSUE TITLE
Failure to connect

### ISSUE THREAD

**Issue by WattsC-90** [sogaani/mozassist#5]
Have followed the instructions, new install of webthings on pi4, just get the following, despite several attempts of reboot power off and checking typos.. Definitely right clientid secret.. Any ideas?
```
{
    "error":"unauthorized_client",
    "error_description":"client id unknown",
   "state":"AMXHz3TvobCbymqkpnycClQjSru1ETnUcBXf2f0XGxzTmstFyZufX3_1mfcbtIRyFfQZZBU1kGf_RbFgJY6xeyIYvF3UxmfOmgiiZS8R3peYel9v6s73KIj3TefqS0QWTJfhKTJzGhR3TLqkF_15BWcc_gv7cc_Ncai7hMft8meGoKeySiq6zBjbfhxk83Xf18f9hyU-VAP6UY2Gn1z_fhgOLx9LAsZTABKu-hxNtUFl6ukJl_OoBub2CxpL4elZ-aanQZyTV558NS8SdmF3eAWV1-bkB7s5cZcPEJR8EPkDaIE4hzQahixMULILCO2smz2PFNKsyZXIPqFm6ttl59aP3tbrwKj8VSrPtrJ-MQ7nmkIUKHrb9eljGPQAexUpL3MM73IPWTynuyeDujtSl1-JZsJAhrr9QTayr_dexLafpQlQ9f6VqoBl6TgV_YYv5BE91Nw7pj4TPFV0HgvCJTnWzBG8XbwNoPIlRxxBrJIh8JX_PDwvRlO1vCnB9ckOIsmTIIeAY9gWvxXKdcJ5RmI6TA882s6FE8IFItlgDXaBdtYXhwdldV3W4Iqv1yFWIAczyGwIiTf8"
}
```

**Comment by WattsC-90** [comment]
hello? is this still actively maintained?

**Comment by manueldimitri** [comment]
Hi WattsC-90,

Have you got any luck getting sogaani to reply?

I'm every interested in that project too.

**Comment by WattsC-90** [comment]
@manueldimitri I havent heard anything from this. I assume this project is dead. It wont connect and iv followed the instructions multiple times to make sure it works, so unfortunately my installation of moz webthings has started gathering dust :(

**Comment by manueldimitri** [comment]
@WattsC-90 I've been trying to look for a alternative to connect to google home assistant using moz webthings. Not easy. Do you found another alternative?

I think home-assistant.io is the same concept and actually have integration with google home assistant. I might have a try.

**Comment by WattsC-90** [comment]
yes home-assistant is similar, there was another: https://about.gbridge.io/notice/ its closed, but the source is still available and it was very easy to use/setup for custom devices. 
From my research the webthings is the way that google amazon etc are going as it pulls from the w3c standard.
